src/networking.c

```diff
@@ -2712,7 +2712,7 @@ char *getClientSockname(client *c) {
 
 /* Concatenate a string representing the state of a client in a human
  * readable format, into the sds string 's'. */
-sds catClientInfoString(sds s, client *client) {
+sds catClientInfoStringGeneric(sds s, client *client, bool hide_user_info) {
```

Comment:
`hide_user_info` is misleading, it could mean that we hide the ACL **user** name.

actually, i'll argue that again that i don't see why CLIENT SETNAME is PII, and if it is, then the ACL user name could be too.

Reply:
Ack, client name will not be redacted